qutebrowser/browser/greasemonkey.py

```diff
@@ -116,6 +117,40 @@ def parse(cls, source, filename=None):
             script.includes = ['*']
         return script
 
+    def force_document_end(self):
```

Comment:
nit: That sounds like the name of a method that does something, not that tells you whether you need to do something.

Reply:
Good point! I renamed it to `needs_document_end_workaround` instead.